ajax/libs/wangEditor/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "wangEditor",
+  "version": "2.1.23",
+  "filename": "js/wangEditor.min.js",
+  "description": "wangEditor - 基于javascript和css开发的 Web富文本编辑器 轻量简洁易用开源免费(wangEditor based on javascript and css that makes it lightweight, easy to create simple, free and opensource.)",
```

Comment:
I think we need the description only in English but not simply translation. Ref: https://github.com/cdnjs/cdnjs/pull/7268#r65663096, https://github.com/cdnjs/cdnjs/pull/10722#pullrequestreview-25726513,